### ISSUE TITLE
When I type 'import' in script tag then Vue Language Server throws exception

### ISSUE THREAD

**Issue by ppasieka** [vuejs/vetur#285]
- Platform: Win
- Vetur version: 0.8.5
- VSCode version: 1.13.1

## Problem
When I type 'import' in script tag then Vue Language Server throws exception

![image](https://user-images.githubusercontent.com/2170707/27516888-b2e52f92-59c2-11e7-9ecb-0ff72ecc85aa.png)

```
vetur initialized
[Error - 16:19:45] Request completionItem/resolve failed.
  Message: Request completionItem/resolve failed with message: Debug Failure. 
  Code: -32603 
C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\typescript\lib\typescript.js:3495
            throw e;
            ^

Error: Debug Failure. 
    at tryReuseStructureFromOldProgram (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\typescript\lib\typescript.js:69500:22)
    at Object.createProgram (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\typescript\lib\typescript.js:69263:34)
    at synchronizeHostData (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\typescript\lib\typescript.js:89032:33)
    at Object.getProgram (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\typescript\lib\typescript.js:89133:13)
    at languageServiceIncludesFile (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\vue-language-server\dist\modes\script\javascript.js:439:24)
    at Object.doValidation (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\vue-language-server\dist\modes\script\javascript.js:164:18)
    at C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\vue-language-server\dist\vueServerMain.js:115:43
    at Array.forEach (native)
    at validateTextDocument (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\vue-language-server\dist\vueServerMain.js:113:59)
    at Timeout._onTimeout (C:\Users\Piotr\.vscode\extensions\octref.vetur-0.8.5\client\node_modules\vue-language-server\dist\vueServerMain.js:107:9)
[Info  - 16:19:45] Connection to server got closed. Server will restart.
vetur initialized
```

## Reproducible Case

Start typing 'import' in vue file. It seems that it throws exception only if I start typing on first line inside script tag.

```
<script>
import // <-  just typed
import { compose, map } from 'ramda'
```

but when I type it after already existing imports then sometimes work and sometimes not. (I'm not sure why.... maybe server is already terminated...)
```
<script>
import { compose, map } from 'ramda'
import // <-  just typed
```

if it helps here is my repo:
https://github.com/ppasieka/calendar

**Comment by HerringtonDarkholme** [comment]
This should be the same issue as 
https://github.com/Microsoft/TypeScript/issues/16591

**Comment by octref** [comment]
Should be fixed by #310.
